### PR TITLE
PubSub example for PHP

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

**Comment by bshaffer** [comment]
hopefully addresses the main concerns of #13

**Comment by bshaffer** [comment]
@tmatsuo PTAL also :)

**Comment by bshaffer** [comment]
bump

File: pubsub/src/DatastoreHelper.php [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+ */
+class DatastoreHelper
+{
+    /** @var Google_Client $client */
+    protected $client;
+
+    /** @var string $datasetId */
+    protected $datasetId;
+
+    public function __construct(\Google_Client $client, $datasetId)
```
Comment (SurferJeffAtGoogle) [inline pubsub/src/DatastoreHelper.php]: The datasetId is usually the same as the project id, right?  We should note that in a comment.

File: pubsub/src/DatastoreHelper.php [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        $this->datasetId = $datasetId;
+    }
+
+    /**
+     * Creates a query object for pulling the last $limit number of
+     * "PubSubMessage" items
+     */
+    public function createQuery($limit = 20)
+    {
+        $request = new \Google_Service_Datastore_RunQueryRequest();
```
Comment (SurferJeffAtGoogle) [inline pubsub/src/DatastoreHelper.php]: Add a comment with the GQL query here.

File: pubsub/test/pubsubTest.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        return $app;
+    }
+
+    public function testInitialPage()
+    {
+        $client = $this->createClient();
+        $crawler = $client->request('GET', '/');
+
+        $this->assertTrue($client->getResponse()->isOk());
+    }
```
Comment (SurferJeffAtGoogle) [inline pubsub/test/pubsubTest.php]: This test hardly covers any code.  Can you add tests that click on the buttons?

**Comment by bshaffer** [comment]
ok @SurferJeffAtGoogle, PTAL

File: pubsub/README.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+gcloud alpha pubsub subscriptions create [your-subscription-name] \
+ --topic [your-topic-name] \
+ --push-endpoint https://[your-project-id].appspot.com/receive_message \
+ –-ack-deadline 30
+```
+
+## Deploy the application to App Engine
+
+```
+$ gcloud preview app deploy app.yaml --set-default --project YOUR_PROJECT_NAME
```
Comment (SurferJeffAtGoogle) [inline pubsub/README.md]: Is this the same string we call [your-project-id] above?  Then we should probably call it YOUR_PROJECT_ID here.

**Comment by SurferJeffAtGoogle** [comment]
LGTM.

File: pubsub/README.md (hunk 4 above)
Comment (bshaffer) [inline pubsub/README.md]: I changed everything to `your-project-name` instead

**Pull request merged**